src/interfaces/permissions/permissions-request.ts

```diff
@@ -1,32 +1,38 @@
 import { DIDResolver } from '../../did/did-resolver';
 import type { Ability, Conditions } from './permission';
+
+import type { JwsFlattened } from '../../jose/jws';
+import type { MessageStore } from '../../store/message-store';
 import type { PermissionsMethod } from './types';
-import type { JwsFlattened }from '../../jose/jws';
 
 /**
  * TODO: add documentation
  * @param message
  */
 export async function PermissionsRequest(
```

Comment:
happy to change the function name to `handlePermissionsRequest`. For context, the reason this function name doesn't start with a verb and is capitalized is because i was looking to match DWN interface method names 1:1 - [example](https://identity.foundation/decentralized-web-node/spec/#request); which allows us to do [this](https://github.com/TBD54566975/dwn-sdk-js/blob/message-store/src/dwn.ts#L29-L30). 

We can still achieve this by adjusting [this methods map](https://github.com/TBD54566975/dwn-sdk-js/blob/message-store/src/dwn.ts#L11).

with this context, are you still of the same opinion?

Reply:
Yeah, still of the same opinion. Happy to chat further!